### ISSUE TITLE
reconnect to new hubServerUrl 

### ISSUE THREAD

**Issue by behnam-basketasia** [dsuryd/dotnetify-react-demo-vs2017#32]
in my project, i set hubServerUrlat the main.js, but i want ping many server and then i want to connect to best server, i need to disconnect from previous hubServerUrl and then connect to new hubServerUrl. how can i handle this changing hubServerUrl? it is possible to change it base on an event?

**Comment by dsuryd** [comment]
It's not currently set up for your use case.  Can you just determine the best server before making the initial connect?

**Comment by behnam-basketasia** [comment]
yes i cant, but my purpose is that i want to check list of servers that get from Backend in an interval and then check best server base on user internet speed and want to change the connection, is there any recommendation for solving this problem? or any update to handle it?
